src/toast/ops/polyfilter/polyfilter.py

```diff
@@ -65,11 +65,14 @@ class PolyFilter2D(Operator):
     )
 
     det_flag_mask = Int(
-        defaults.det_mask_invalid | defaults.det_mask_processing,
+        defaults.det_mask_invalid,
```

Comment:
Why were the processing flags dropped here? The polyfilter seems like an ideal application for them.

Reply:
Since a filter modifies the data, any samples that are **not** modified become invalid (assuming the notes earlier in this PR discussion).  So if a filter uses the processing flag, then on output those unfiltered samples will be flagged invalid.  (Thinking out loud) I guess this makes sense.  If you are applying a filter that does not make sense to apply across a turnaround (for example), then you really are never going to use the turnarounds later and it makes sense that they are invalid.  If we have an operator applying some other filter to an entire observation (for example timeconstant deconvolution), then the default would be to consider all valid data.  So this choice of defaults is operator-dependent, and I agree for the polynomial filters it makes sense to include the processing mask.

Reply:
I have gone through every operator and tried to set the defaults to sensible ones:

- if an operator is simulating data, it only uses the `invalid` mask and tries to fill as many samples as possible
- if an operator is doing low-level filtering of data that is independent of any sky / processing mask, then it uses the `invalid` bitmask.
- if an operator is doing higher-level filtering, then it also uses the `processing` bit.
- if an operator is doing high-level mapmaking operations, it uses the `nonscience` compound mask.

Obviously these are just the defaults, and can be overridden on a case by case basis.  For example when making planet maps you would not set the SSO bit.